source/fundamentals/linq.txt

```diff
@@ -1,8 +1,8 @@
 .. _csharp-linq:
 
-====
-LINQ
-====
+================================
+Language Integrated Query (LINQ)
```

Comment:
good thought, but it's [inconsistent with the style guide.](https://www.mongodb.com/docs/meta/style-guide/style/abbreviations/#:~:text=Don%27t%20show%20both%20the%20spelled%2Dout%20term%20and%20its%20abbreviation%20in%20a%20title%20or%20heading.%20In%20most%20cases%2C%20use%20the%20abbreviation%20in%20the%20title%20or%20heading%20and%20show%20the%20spelled%2Dout%20term%20and%20its%20abbreviation%20on%20first%20use%20in%20the%20text.) how about:

```suggestion
LINQ Syntax for Aggregation Operations
```

Reply:
Thanks for that style guide link; yes LGTM!